### ISSUE TITLE
Cluster Elefante Bornéu [3x VPS][CPU: 3][RAM: 6GB][Disk: 60GB][Cost: 10.05 USD/Month]

### ISSUE THREAD

**Issue by fititnt** [EticaAI/forum#81]
![Borneo Elephants](https://user-images.githubusercontent.com/812299/59556732-fedc5200-8f9d-11e9-88b6-14d40888b778.jpg)

Wikipedia: [Borneo pygmy elephant](https://en.wikipedia.org/wiki/Borneo_elephant) \| Image source: [Wikipedia Commons](https://commons.wikimedia.org/wiki/File:Borneoelephants.jpg)

----

- Creation of the first node: 2019-05-26
- Deactivation of the last node: 
- Total cost: **10.05 USD/Month** (3x 3.35 USD/month)
    - Paid by @fititnt

## Extra information

- Nodes:
    - All Nodes (inactive & Active) of Elefante Bornéu: <https://github.com/EticaAI/forum/issues?q=label%3Acluster-elefante-borneu>
    - [Etica.AI Infrastructure: Clusters & VPS: https://github.com/orgs/EticaAI/projects/2](https://github.com/orgs/EticaAI/projects/2)

- Main related issues
     - <em lang="pt">Servidores em cluster Elefante Bornéu https://github.com/fititnt/cplp-aiops/issues/45 <sup>Portuguese</sup></em>
- Main related repositories
     - <em lang="pt">[AIOps para Comunidade dos Países de Língua Portuguesa, 2019/01](https://github.com/fititnt/cplp-aiops) <sup>Portuguese</sup></em>

**Comment by fititnt** [comment]
Detailed explanations:

- https://github.com/fititnt/cplp-aiops/issues/45#issuecomment-506936177 <sup>Portuguese</sup> 
- https://github.com/fititnt/cplp-aiops/issues/5#issuecomment-501605356 <sup>Portuguese</sup>

----

For more details, please read the links with detailed explanations. Ok to use automatic translation.

TL;DR: **We found much more cheaper datacenter to host our VPSs in Germany, Europe. Using Tsuru Paas (https://github.com/EticaAI/forum/issues/89) on the Cluster Águia Pescadora (https://github.com/EticaAI/forum/issues/82) could be possible at some point to host the Cluster Elefante Bornéu (witch the idea was about have redundant data storage in high availability).**

Quick note for who will not follow all the links: even if we could (in fact, we had already an MVP on North America) have at least a Galera Cluster (MySQL in High Availability) ready to use, as first semester of 2019 both price per online VPS and need for this level of fault tolerance was less need than other features.

Compare the prices of the full cluster on North America vs a single machine on Europe
- Cluster Elefante Bornéu [3x VPS][CPU: 3][RAM: 6GB][Disk: 60GB][Cost: 10.05 USD/Month]
- VPS Águia Pescadora Delta [6vCPU / 16GB RAM / 400GB] [8.99 EUR/month (*)]

For who live in the Americas (me included) the latency is obviously less desirable, but for the same price is possible to get around 2x CPUs, 3x RAM & 6x SSD Disk. Also strategies to make easier to share resources (like try to otimize some PaaS) could be more useful in the short term.

Anyway all work preparing the cluster (see https://github.com/fititnt/cplp-aiops/tree/master/logbook & https://github.com/fititnt/cplp-aiops/issues?utf8=%E2%9C%93&q=label%3Aserver-elefante-borneu) still documented for who is interested in create a Galera Cluster direct on VPSs and not inside some PaaS or docker conteiners.
